libs/dao/src/main/java/com/akto/dao/MCollection.java

```diff
@@ -91,18 +91,6 @@ public<V, U> List<T> findAll(String key1, V value1, String key2, U value2) {
         return this.findAll(and(eq(key1, value1), eq(key2, value2)));
     }
 
-    public<V> List<T> findAll(String key, Collection<V> values) {
-        MongoCursor<T> cursor = this.getMCollection().find(in(key, values)).cursor();
-
-        ArrayList<T> ret = new ArrayList<T>();
-
-        while(cursor.hasNext()) {
```

Comment:
check for instances

Reply:
yup, compiles, so should not be a problem